oca_dependencies.txt

```diff
@@ -4,3 +4,4 @@ account-invoicing
 server-tools
 stock-logistics-workflow
 sale-workflow
+purchase-workflow https://github.com/Eficent/purchase-workflow 10.0-purchase_order_block
```

Comment:
Do not forget to remove this, I would put into a different commit to be always aware.